config/routes.rb

```diff
@@ -95,9 +95,18 @@
       end
     end
 
+    resources :spaces do
+      get 'index', path: '/', on: :collection
+      get 'edit', on: :member
```

Comment:
Do you actually need these? I though `resources :spaces` would create an index and an edit action. Also, if you aren't using some of the actions you should have `resources: :spaces, only: [...]` like here https://github.com/uOttawa-Makerspace/MakerSpaceRepo/blob/aec4840fd1dca34e9341c118fbbe6800d40498fc/config/routes.rb#L78